css/style.css

```diff
@@ -311,23 +309,16 @@ a.footer-social:hover {
     font-size: 31px;
   }
 
-  .footer1 {
-    position: sticky;
-    bottom: 0;
-    padding: 30px 0;
-    border: 1px solid #ff8008;
-    background-color: #ff8008;
-    display: flex;
-    justify-content: center;
-    line-height: var(--space-xl);
-  }
-
-  .footer-social a {
+  .footer-social  {
     display: flex;
     justify-content: center;
     text-decoration: none;
     color: var(--text-color);
   }
```

Comment:
You want to apply all this flex stuff to the surrounding div, not the anchor element.